.github/ISSUE_TEMPLATE/report_documentation_issue.yml

```
@@ -0,0 +1,80 @@
+            </picture>
+          </a>
+        </p>
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for contributing to improve Dagster.
+        If you have a change ready to submit, you do not need to create an issue.
+        Instead, you can open a [pull request](https://github.com/dagster-io/dagster/pulls).
+  - type: markdown
+    attributes:
+      value: >
+        Writer's Guide: [Docs README](https://github.com/dagster-io/dagster/tree/master/docs)
+  - type: textarea
+    attributes:
+      label: What content is missing
+      description: Help us to understand your request in context.
+      placeholder: Please provide a brief description of the missing content.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Type of content
+      description: What type would the content most likely belong to?
+      multiple: true
+      options:
+        - "Tutorial"
+        - "Main Concept: 'What is X in Dagster' where X is a Dagster concept."
+        - "Integration Guide: 'How to use X in Dagster' where X is a 3rd-party package/tool, e.g. dbt, Spark, etc"
+        - "Best Practices Guide: 'How to do X in Dagster' where X is usually a use case and can be addressed by incorporating multiple Dagster concepts at a time."
+        - "Deployment Guide: 'How to deploy Dagster to X', e.g. Docker, K8S, GCP, etc"
+        - "API Reference: e.g. docstring"
+        - "Other"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional information
+      description: Anything else we need to know?
+      placeholder: Enter additional details that may be relevant.
+  - type: checkboxes
+    attributes:
+      label: Are you willing to submit PR?
+      description: >
+        This is not required, but we are happy to guide you through the contribution process.
+        [Find us on Slack](https://dagster.io/slack)!
+      options:
+        - label: I am willing to submit a PR!
+  - type: checkboxes
+    attributes:
+      label: Code of Conduct
+      description: We require that everyone agrees to our Code of Conduct.
+      options:
+        - label: >
+            I agree to follow Dagster's
+            [Code of Conduct](https://github.com/dagster-io/dagster/blob/master/CODE_OF_CONDUCT.md).
+          required: true
+  - type: markdown
+    attributes:
+      value: "# Thanks for contributing to Dagster!"
+  - type: textarea
+    attributes:
+      label: Message from the maintainers
+      description: This form field should be ignored. This is to include a footer message on the generated issue.
+      value: |
+        This form field should be ignored.
+
+        Impacted by this bug? Give it a 👍. We factor engagement into prioritization.
```

```suggestion
        Impacted by this issue? Give it a 👍.! We factor engagement into prioritization.
```

Changing `bug > issue`, since reports aren't always bugs.